docs/architecture/psa-shared-memory.md

```diff
@@ -661,7 +661,7 @@ psa_status_t mem_poison_psa_aead_update(psa_aead_operation_t *operation,
 #define psa_aead_update(...) mem_poison_psa_aead_update(__VA_ARGS__)
 ```
 
-There now exists a more generic mechanism for making exactly this kind of transformation - the PSA test wrappers, which exist in the files `tests/include/test/psa_test_wrappers.h` and `tests/src/psa_test_wrappers.c`. These are wrappers around all PSA functions that allow testing code to be inserted at the start and end of a PSA function call.
+There now exists a more generic mechanism for making exactly this kind of transformation - the PSA test wrappers, which exist in the files `framework/tests/include/test/psa_test_wrappers.h` and `framework/tests/src/psa_test_wrappers.c`. These are wrappers around all PSA functions that allow testing code to be inserted at the start and end of a PSA function call.
```

Comment:
Aren't the wrappers staying in mbedtls?

Reply:
I expect the wrapper generation code to be in the framework. The generated wrappers should eventually not be in the repository at all, but in the meantime (until we adapt the build scripts to cope with generated header files), the wrappers belong in `mbedtls/mbedtls-3.6` and `TF-PSA-Crypto/main`.